lib/JSON/Validator/OpenAPI.pm

```diff
@@ -282,10 +285,9 @@ sub _coerce_by_collection_format {
   return $single ? $data->[0] : $data;
 }
 
-sub _invalid_in {
-  my ($self, $in) = @_;
+sub _confess_invalid_in {
```

Comment:
Just made this into a function, to make it stand out. Nothing to do with this PR really.